torchtext/csrc/core/vocabulary.h

```diff
@@ -0,0 +1,36 @@
+#ifndef VOCABULARY_H
+#define VOCABULARY_H
+
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+namespace torchtext {
+namespace core {
+class Vocabulary {
+  static const size_t MAX_VOCAB_SIZE = 30000000;
+  static const size_t negative = static_cast<size_t>(-1);
```

Comment:
Why is this necessary?

Reply:
Do you mean negative? I'm using ```size_t``` for indexes which is unsigned and indexes must initially be negative(-1). So this is represents negative in unsigned integer (std::string uses the same method for negative).